### PR TITLE
repair interface with psi4

### PR DESCRIPTION
**Pull request by loriab**

### CODE REVIEW

File: src/efp.c [hunk 1]
```diff
@@ -1991,6 +1991,27 @@ efp_create(void)
...
+{
+    assert(efp);
+
+    efp->get_electron_density_field = fn;
+
+    return EFP_RESULT_SUCCESS;
+}
+
+EFP_EXPORT enum efp_result
+efp_set_electron_density_field_user_data(struct efp *efp, void *user_data)
```
Comment (loriab) [inline src/efp.c:2006]: fwiw, I was never sure what was supposed to go in this function, so Psi4 doesn't use it.

**Review by loriab — COMMENTED** [review]
Tested against (1) local pylibefp `test_scf.py::test_qmefp` at #18 and (2) a local dev psi4: all efpefp and qmefp tests and all pass cleanly against a libefp with this patch. No downstream edits required.

Thanks very much, I vote merge!

**Comment by loriab** [comment]
Maybe this one next, please, @slipchenko . Then #18 will be a cleaner changeset.

**Comment by slipchenko** [comment]
Uff, this is a relief! Thanks!

**Comment by loriab** [comment]
Thanks! I'll work on #18 tomorrow.

**Pull request merged**